README.md

```diff
@@ -1,29 +1,29 @@
 Mycroft [![Build Status](https://travis-ci.org/MycroftAI/mycroft-core.svg?branch=master)](https://travis-ci.org/MycroftAI/mycroft-core) [![Coverage Status](https://coveralls.io/repos/github/MycroftAI/mycroft-core/badge.svg?branch=dev)](https://coveralls.io/github/MycroftAI/mycroft-core?branch=dev)
 ==========
 
-NOTE: The default branch for this repo is dev. This should be considered a working beta. If you want to clone a more stable version, switch over to the 'master' branch.  
+NOTE: The default branch for this repository is 'dev', which should be considered a working beta. If you want to clone a more stable version, switch over to the 'master' branch.  
 
-Full docs at: https://docs.mycroft.ai
+Full documentation at: https://docs.mycroft.ai
 
 Release notes at: https://docs.mycroft.ai/release-notes
 
 Pair Mycroft instance at: https://home.mycroft.ai
 
-Join the Mycroft Slack(chat) channel: http://mycroft-ai-slack-invite.herokuapp.com/
+Mycroft Slack (chat) channel: http://mycroft-ai-slack-invite.herokuapp.com/
```

Comment:
Let's switch to https://mycroft.ai/to/slack.  Later we can redirect this link to point to Mattermost instead of leading them to a dead Slack.